docs/pages/project/contributing/contributing-cli.md

```diff
@@ -128,6 +128,27 @@ When designing for the command line interface, ask and consider the following qu
 
 You will need to provide a short and long description of the command for the help pages and also for the Meshery Documentation.
 
+## Refactoring Mesheryctl Code
+
+In the latest update of Meshery, the meshconfig has been refactored to have getters and setters which can further be used to manipulate any config, making it more easier to perform specific functions. 
+
+A central struct has been configured to contain the Meshery Config File, where all the operations to manipulate the config into a central struct with getters and setters have been defined. A central function has also been added to write the config.yaml file.
```

Comment:
Please be consistent in the references to the "Meshery Config File". You used "meshconfig" above. Let's stick with that. Is there a spot anywhere on this page that explains what `meshconfig` is? If not, an explanation is a good use of the `include alert.html`